package.json

```diff
@@ -13,12 +13,11 @@
     "watch:babel": "npm run build:babel -- -w",
     "watch:webpack": "webpack --progress --profile --colors --watch",
     "server": "node start.js",
-    "test": "npm-run-all --parallel test:server test:http test:depcheck test:lint test:karma",
+    "test": "npm-run-all --parallel test:http test:depcheck test:lint test:karma",
     "test:karma": "karma start",
     "test:coverage": "KARMA_ENV=coverage karma start --reporters coverage,coveralls",
     "watch:test": "karma start --no-single-run --browsers Firefox",
-    "test:http": "mocha tests/http/*.test.js",
-    "test:server": "node tests/start.js",
```

Comment:
Thoughts on removing this script?

Reply:
It literally did nothing, AFAICT started up, and turned off the server. we can leave that testing to the Hapi test suite.

Reply:
iirc it made it so we would catch it sooner? Best reason I could come up with.

Reply:
Our test suite starts a server before each new case, so having this test is pretty redundant still